packages/f/fontsource-abeezee.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "fontsource-abeezee",
+  "description": "Self-host the ABeeZee font in a neatly bundled NPM package.",
+  "main": "index.css",
+  "keywords": [
+    "fontsource",
+    "font",
+    "font family",
+    "google fonts",
+    "abeezee",
+    "ABeeZee",
+    "css",
+    "sass",
+    "front-end",
+    "web",
+    "typeface",
+    "variable"
+  ],
+  "author": "Google Inc.",
```

```suggestion
  "authors": [
    {
      "name": "Google Inc."
    }
  ],
```
We take an `authors` array rather than a single `author`